testfiles/gedcom70/maximal70.ged

```diff
@@ -2,6 +2,15 @@
 1 GEDC
 2 VERS 7.0
 1 NOTE This file is intended to provide coverage of parts of the specification and does not contain meaningful historical or genealogical data.
+2 CONT That observation applies even to the substructures of this NOTE.
```

Comment:
```suggestion
2 CONT That observation applies even to the substructures of this NOTE; for example, the TRAN does not translate this text, it merely exercises the presence of a TRAN.
```

Reply:
Another possible translatable phrase: "See the specification for advice on how to use" / "...advise"